src/components/organisms/UiHorizontalPaging/UiHorizontalPaging.spec.ts

```diff
@@ -1,12 +1,5 @@
 import { mount } from '@vue/test-utils';
-import {
-  nextTick,
-  h,
-} from 'vue';
-import UiHeading from '@/components/atoms/UiHeading/UiHeading.vue';
-import UiMenuItem from '@/components/organisms/UiMenu/_internal/UiMenuItem.vue';
-import UiHorizontalPaging from './UiHorizontalPaging.vue';
-import UiHorizontalPagingItem from './_internal/UiHorizontalPagingItem.vue';
+import UiHorizontalPaging from './_.vue';
```

Comment:
What is `_.vue` 🤔

Reply:
WebStorm auto refactor. I miss this auto-changes. Thanks.